package.json

```diff
@@ -42,7 +42,6 @@
     "dockerfile-ast": "^0.5.0",
     "dotenv": "^16.0.2",
     "express-openid-connect": "^2.8.0",
-    "firecracker-client": "./deps/firecracker-client",
```

Comment:
Please remove all dependencies which were required by the firecracker client/integration.